sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Tag.java

```diff
@@ -18,6 +19,7 @@
     @Column(name = "tagName")
     private String name;
 
+    @JsonIgnore
```

Comment:
Please let's use Jackson annotations only for DTO level. We can use `TagDto`

Reply:
done